tests/wpt/web-platform-tests/html/syntax/serializing-html-fragments/serializing.html

```diff
@@ -25,6 +25,7 @@
 <span>&quot;</span>
 <span><style><&></style></span>
 <span><script type="test"><&></script></span>
+<script type="test"><&></script>
```

Comment:
No need for the `type="test"` here, since we only care about the child contents of the element.